src/main/resources/application.properties

```diff
@@ -1 +1,4 @@
 spring.application.name=GitHubAiAPI
+
+
+Spring.demo.value= test
```

Comment:
It is a good practice to use meaningful names for your properties. 'Spring.demo.value' does not clearly indicate its purpose. Consider renaming it to something more descriptive.